### PR TITLE
Remove Docs from Master

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
`docs/improbadoc-conversion` will be where the docs will live (branch name will change!) and the website is the user facing docs. 
Moved coding standards to the root directory.
Deleted `./docs/` and got rid of all those heavy GIFs.
Repointed all links.

Second commit is the important one!

#### Tests
Ran DocsLinter
#### Documentation
THERE ARE NO DOCS
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
@ElleEss

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,25 +8,24 @@ The SpatialOS Game Development Kit (GDK) for Unity enables game developers to bu
...
+
+* [Setup Guide](https://docs.improbable.io/unity/alpha/setup-and-installing#set-up-your-machine)
+* [Documentation](https://docs.improbable.io/unity/alpha/)
 * [Development Roadmap](https://trello.com/b/29tMKyQC)
-* [FAQs](docs/content/faqs/faqs.md)
 * [Discord](https://discord.gg/SCZTCYm)
 * [Forums](https://forums.improbable.io/latest?tags=unity-gdk)
 * [Github Issues](https://github.com/spatialos/UnityGDK/issues)
-* [Known Issues](docs/known-issues.md)
+* [Known Issues](https://docs.improbable.io/unity/alpha/known-issues)
```
Comment (jared-improbable) [inline README.md]: No "GDK for Unity" reference in this Url?

Comment (jamiebrynes7) [reply README.md]: There is not!

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -11,13 +11,16 @@ later releases and we’ll keep you posted on this). However, we are taking issu
...
 
 ## Coding standards
-See the [Unity GDK C# coding standards guide](../docs/contributions/unity-gdk-coding-standards.md).
+
+See the [Unity GDK C# coding standards guide](../coding-standards.md).
 
 ## Getting started
-See [About the SpatialOS Unity GDK](../docs/README.md) for
+
+See [About the SpatialOS Unity GDK](../README.md) for
```
Comment (ElleEss) [inline .github/CONTRIBUTING.md]: Do you think this should link to the Get Started website doc?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -11,13 +11,16 @@ later releases and we’ll keep you posted on this). However, we are taking issu
 See the [Contributions guide](https://improbableio.atlassian.net/wiki/x/foDrDw).
 
 ## Coding standards
-See the [Unity GDK C# coding standards guide](../docs/contributions/unity-gdk-coding-standards.md).
+
+See the [Unity GDK C# coding standards guide](../coding-standards.md).
```
Comment (ElleEss) [inline .github/CONTRIBUTING.md]: GDK for Unity

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -11,13 +11,16 @@ later releases and we’ll keep you posted on this). However, we are taking issu
...
 
 ## Getting started
-See [About the SpatialOS Unity GDK](../docs/README.md) for
+
+See [About the SpatialOS Unity GDK](../README.md) for
  details on how to get started using the Unity GDK.
 
 ## Warning
```
Comment (ElleEss) [inline .github/CONTRIBUTING.md]: We are blasting this warning away.

**Pull request merged**